src/config/routes.ts

```diff
@@ -1,6 +1,7 @@
 export const AppRoutes = {
   '404': '/404',
   welcome: '/welcome',
+  welcomeLp: '/welcome-lp',
```

Comment:
/welcome/social-login perhaps?

Reply:
This would require us to move the welcome page into a directory and rename all occurences of `AppRoutes.welcome` to `AppRoutes.welcome.index` though.

Reply:
I'm aware.